test/unit_tests/test_config_data_model_generator.py

```diff
@@ -0,0 +1,48 @@
+import sys
+
+from exasol_script_languages_container_ci.lib.config.data_model_generator import CONFIG_DATA_MODEL_FILE_NAME, \
+    generate_config_data_model
+
+
+def test_loading_generated_module(tmp_path):
+    config_data_model_file = tmp_path / CONFIG_DATA_MODEL_FILE_NAME
+    generate_config_data_model(config_data_model_file)
+    module = load_module(config_data_model_file)
+    assert True
```

Comment:
Add a real assertion

Reply:
the load module is more or less the actual assert, but I added a bit better assert, anyway